lang/Cargo.toml

```diff
@@ -59,6 +59,5 @@ borsh = ">=0.9, <0.11"
 bytemuck = "1"
 solana-program = "1.16"
 thiserror = "1"
-
-# TODO: Remove. This crate has been added to fix a build error with the 1.16.0 release.
-getrandom = { version = "0.2", features = ["custom"] }
```

Comment:
No need to remove this for now as it will be removed when we stop the `1.16` support.